### ISSUE TITLE
KeyError: 'scene_coords_half'

### ISSUE THREAD

**Issue by Doraemon167** [zhenpeiyang/FvOR#4]
Dear zhenpeiyang，
      Thanks for you work! When I was testing FvOR pose estimation model, I had a problem that KeyError: 'scene_coords_half'. So do you have some advice? @zhenpeiyang  Thanks.

**Comment by zhenpeiyang** [comment]
Hi @Doraemon167 , I did not encounter your issue after testing the setup on a new machine. Did you download the processed data from both OccNet and FvOR?

**Comment by Doraemon167** [comment]
Yes, I downloaded the processed data from both OccNet and FvOR，when I was tested the FvOR recon model trained with Ground Truth camera poses is norm. But test the FvOR pose estimation model will encounter this error. @zhenpeiyang

**Comment by Doraemon167** [comment]
Hello， @zhenpeiyang . I solved it. My Pytorch version is 1.11, so I edited some source code. I clone this projector again and using Pytorch of 1.7. this error was solved. Thanks!

**Comment by Doraemon167** [comment]
Hello， @zhenpeiyang . I solved it. My Pytorch version is 1.11, so I edited some source code. I clone this projector again and using Pytorch of 1.7. this error was solved. Thanks!

**Comment by Doraemon167** [comment]
Hello, @zhenpeiyang , I have another question, When we test the FvOR recon model trained with Ground Truth camera poses that must  provide camera.npz?

**Comment by zhenpeiyang** [comment]
Hi @Doraemon167, sorry for the late reply. Yes the camera.npz is needed.
